packages/web-app-files/src/components/SideBar/TagsPanel.vue

```diff
@@ -0,0 +1,158 @@
+<template>
```

Comment:
Small challenge from me: Since this is an entirely new component I would love to see it only using composition api. What do you guys think?